crates/biome_configuration/src/linter/rules.rs

```diff
@@ -910,8 +999,8 @@ impl A11y {
         index_set
     }
     #[doc = r" Checks if, given a rule name, matches one of the rules contained in this category"]
-    pub(crate) fn has_rule(rule_name: &str) -> bool {
-        Self::GROUP_RULES.contains(&rule_name)
+    pub(crate) fn has_rule(rule_name: &str) -> Option<&'static str> {
+        Some(Self::GROUP_RULES[Self::GROUP_RULES.binary_search(&rule_name).ok()?])
```

Comment:
What measures do we have to ensure that groups stay sorted?

Reply:
Good point. It is something I forgot to check.

Reply:
I added tests at the end of the file.